tests/Test 1 - Perform Login.spec.js

```diff
@@ -0,0 +1,22 @@
+const {test, expect } = require('playwright/test')
+
+//import { test, expect } from '@playwright/test';
+
+test('Perform login, Verify shopping cart and list of products are displaye', async ({ page }) => {
```

Comment:
I would suggest to have describe (as a test suite) and login actions inside in corresponding beforeEach hook. You could recheck with yesterdays call, we discussed it.

Reply:
fixed